pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -579,15 +589,17 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 						forwardPort.Protocol = "TCP"
 					}
 
-					if _, ok := portForwardMap[fmt.Sprintf("%s-%d", forwardPort.Protocol, forwardPort.Port)]; ok {
-						causes = append(causes, metav1.StatusCause{
-							Type:    metav1.CauseTypeFieldValueInvalid,
-							Message: fmt.Sprintf("Port and protocol combination must be unique"),
-							Field:   field.Child("domain", "devices", "interfaces").Index(idx).Child("ports").Index(portIdx).String(),
-						})
-					}
+					if forwardPort.Name != "" {
+						if _, ok := portForwardMap[forwardPort.Name]; ok {
+							causes = append(causes, metav1.StatusCause{
+								Type:    metav1.CauseTypeFieldValueDuplicate,
+								Message: fmt.Sprintf("Duplicate value: %s", forwardPort.Name),
```

Comment:
it could also indicate it's a duplicate "name of the port" and not just a vague "value"